linode/manager
# refactor: [M3-6522] - BetaChip V7 story migration and cleanup

## Description 📝
Small PR to migrate the BetaChip component story to a V7 (`.tsx`) story 

## Changes  🔄
- Clean up `<BetaChip />` (props, export, styling)
- New v7 `.tsx` story to replace the `.mdx` one
- Update storybook to `7.5.2`

## Preview 📷
This PR should not introduce any visual change

## How to test 🧪

### Prerequisites
- pull code locally
- run `yarn && yarn storybook`

### Verification steps 
- navigate to http://localhost:6006/?path=/docs/components-chip-betachip--documentation
- confirm story loads properly
- start cloud manager and confirm there's no regression for the beta chip

## As an Author I have considered 🤔

*Check all that apply*
